src/odemis/cli/test/main_test.py

```diff
@@ -313,7 +309,7 @@ def test_stop(self):
         self.assertEqual(ret, 0, "trying to run '%s'" % cmdline)
     
     def test_acquire(self):
-        picture_name = "test.tiff"
+        picture_name = "TÈßt.tiff"
```

Comment:
Could you add a comment like "utf-8 bytes on Python 2, unicode on Python 3", to make the trick clear.